lib/manager/util/transform.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import toSentenceCase from '../../common/util/to-sentence-case'
+
+import type {FeedTransformation, FeedVersion} from '../../types'
+
+/**
+ * Returns human readable name for transformation type.
+ */
+export function getTransformationName (
+  type: string,
+  transformation?: FeedTransformation,
+  versions?: Array<FeedVersion>
+) {
+  let name = type
+    // Remove transformation from name.
+    .replace('Transformation', '')
+    // Regex finds/captures words in camel case string and splits camel-cased words.
+    // Derived from: https://stackoverflow.com/a/18379358/915811
+    .replace(/([a-z])([A-Z])/g, '$1 $2')
+  name = toSentenceCase(name)
```

Comment:
Too fancy and complex. Also, not sure how well this would work with i18n. There's got to be a way to just use a lookup of transformation type to transformation human-readable name.